packages/ui/client/components/views/ViewConsoleOutputEntry.cy.tsx

```diff
@@ -0,0 +1,42 @@
+import Filter from 'ansi-to-html'
+import ViewConsoleOutputEntry from './ViewConsoleOutputEntry.vue'
+
+const htmlSelector = '[data-type=html]'
+const textSelector = '[data-type=text]'
+
+describe('ViewConsoleOutputEntry', () => {
+  it('test plain entry', () => {
+    const content = new Date().toISOString()
+    const container = cy.mount(
+      <ViewConsoleOutputEntry
+        task-name="test/text"
+        type="stderr"
+        time={Date.now()}
+        html={false}
+        content={content}
+      />,
+    ).get(textSelector)
+    container.should('exist')
+    container.invoke('text').then((t) => {
+      expect(t, 'the message has the correct message').equals(content)
+    })
+  })
+  it('test html entry', () => {
+    const now = new Date().toISOString()
+    const content = new Filter().toHtml(`\x1B[33m${now}\x1B[0m`)
+    const container = cy.mount(
+      <ViewConsoleOutputEntry
+        task-name="test/html"
+        type="stderr"
+        time={Date.now()}
+        html={true}
+        content={content}
+      />,
+    ).get(htmlSelector)
+    container.should('exist')
+    container.children('span').then((c) => {
+      expect(c, 'the message has the correct message').to.have.text(now)
+      expect(c, 'the message has the correct text color').to.have.attr('style', 'color:#A50')
+    })
+  })
```

Comment:
This test is brittle because it depends on the specific CSS output of UnoCSS on styles 😮. 

Try to avoid these kinds of tests.

```suggestion
      <ViewConsoleOutputEntry
        task-name="test/html"
        type="stderr"
        time={Date.now()}
        html={true}
        content={content}
      />,
    )
    cy.get(htmlSelector)
      .should('have.class', 'text-red-600')
      .and('be.visible')
      .and('contain.text', now)
  })
```

Reply:
@JessicaSachs 

The resulting html is this:
```html
<pre data-type="html"><span style="color:#A50">2022-04-07T19:10:53.637Z</span></pre>
```
using your suggestion, the messages aren't totally accurate (the `pre` doesn't have that style, it's the inner `span` that does):

![imagen](https://user-images.githubusercontent.com/6311119/162279440-8cadeb86-9500-458b-8af0-e83f14a34220.png)